legacy/expressions/context.go

```diff
@@ -294,8 +294,8 @@ func newMigrationVars() map[string]interface{} {
 		"step": &varMapper{
 			substitutions: map[string]interface{}{
 				"contact":     contact,
-				"__default__": "input",
-				"value":       "input",
+				"__default__": "input.text",
+				"value":       `input.text`,
```

Comment:
So this isn't quite right for anyone who is checking for attachments, but not sure what to do. In legacy expressions `@step` and `@step.value` are the text and the attachment URLs joined by `\n` Replicating that in an expression would require a crazy expression, and then the operand for every switch router would be a crazy expression. Not sure if the new editor would even recognize those as "wait for response".

So maybe we need to add a key to the new `@input` dict that replicates that?

Reply:
Does kinda suck that out story for checking for the presence of attachments is checking if `@step.value` contains "http"